### PR TITLE
Modifications to viewShift command

### PR DESCRIPTION
**Pull request by Gabau**
* viewShift command not displaying the correct results for single date periods. (e.g. 20-01-01 20-01-01)
 * Change viewShift command to take in only one date, and find the next date that is a DAYOFWEEK

### CODE REVIEW

**Review by IrvingHe000 — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -426,15 +427,19 @@ The output will look like the following.
...
+* If DATE is provided, the shift that will be looked at is the next date which satisfies the DAY entry.
 * Similarly, if only one date input is provided, the next occurrence of the provided shift *after the provided date* is displayed.
+* If no input is provided, displays output for the shift occurring now. (The current date and time)
 
 Formats:  
-`viewShift -d DAY-shift_number [da/START_DATE] [da/END_DATE]`  
-`viewShift -ti DAY-HH:mm [da/START_DATE] [da/END_DATE]`
+`viewShift -d DAY-shift_number [da/DATE]`  
+`viewShift -ti DAY-HH:mm [da/DATE]`
+`viewShift`
```
Comment (mweeruien) [inline docs/UserGuide.md:442]: Can we standardize to "DAYOFWEEK"? Other functions use it also, like addShift

**Review by mweeruien — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/logic/commands/ViewShiftCommand.java [hunk 2]
```diff
@@ -31,11 +29,13 @@
...
-            + SHIFT_PERIOD_PARSING_DEFAULT + "\n\n"
+            + "Takes in one optional date input, the shift viewed will be for the next date that shift occurs."
+            + "For instance, the shift viewed given a date of sunday for da/2021-11-06 (a saturday) "
+            + "will be for the date 2021-11-07, the next date. Date input expected in YYYY-MM-DD\n\n"
             + "Parameters:\n"
             + PREFIX_DASH_DAY_SHIFT + " day-slot_number\n"
-            + PREFIX_DASH_TIME + " day-time" + " (time is in format HH:mm)"
-            + DATE_RANGE_INPUT + "\n\n"
+            + PREFIX_DASH_TIME + " day-time" + " (time is in format HH:mm)\n"
+            + "[" + PREFIX_DATE + "DATE]" + "\n\n"
```
Comment (mweeruien) [inline src/main/java/seedu/address/logic/commands/ViewShiftCommand.java:38]: "day-slot_number" etc shld be capitalized for consistency

File: docs/UserGuide.md (hunk 1 above)
Comment (Gabau) [reply docs/UserGuide.md]: Yea. I think that HH:mm does not really give much information of what the field is? Any suggestions on what it should be or maybe use TIME?

**Review by mweeruien — APPROVED** [review]
LGTM!

**Pull request merged**